### PR TITLE
Add Dependabot configuration

### PR DESCRIPTION
**Pull request by b1ron**
I will link issue after review.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+
```
Comment (AlekSi) [inline .github/dependabot.yml:2]:
```suggestion
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+version: 2
+updates:
+
+  # Maintain dependencies for Java
```
Comment (AlekSi) [inline .github/dependabot.yml:6]:
```suggestion
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - package-ecosystem: "maven"
+    directory: "/"
+    assignees: [b1ron]
+    open-pull-requests-limit: 5
+    schedule:
+      # Checks for updates 10 minutes before dance.
```
Comment (AlekSi) [inline .github/dependabot.yml:12]: The only reason that would make sense is if Dependabot's PRs will be approved and merged in 10 minutes – in the middle of the night.

Comment (b1ron) [reply .github/dependabot.yml:12]: Yes understood, makes sense. I will change the time to early in the week.

**Review by AlekSi — APPROVED** [review]

**Pull request merged**